README.md

```diff
@@ -6,58 +6,62 @@
 [![Latest Release](https://img.shields.io/github/v/release/bitovi/bitops)](https://github.com/bitovi/bitops/releases)
 [![Join our Slack](https://img.shields.io/badge/slack-join%20chat-611f69.svg)](https://www.bitovi.com/community/slack?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-BitOps is a way to describe the infrastructure and things deployed onto that infrastructure for multiple environments in a single place called an [Operations Repo](docs/operations-repo-structure.md).
-
-https://bitovi.github.io/bitops/
+### tl;dr
+BitOps is two things; 
+- A automated tool [orchestrator](docs/about.md)
+- A way to describe [infrastructure](docs/operations-repo-structure.md)
```

Comment:
A way to describe [infrastructure](docs/operations-repo-structure.md) for many environments and IaC tools